core/src/test/scala/unit/kafka/server/SaslApiVersionsRequestTest.scala

```diff
@@ -60,14 +60,16 @@ object SaslApiVersionsRequestTest {
 
     List(ClusterConfig.defaultBuilder
       .setSecurityProtocol(securityProtocol)
-      .setTypes(Set(Type.ZK).asJava)
+      .setTypes(Set(Type.KRAFT).asJava)
       .setSaslServerProperties(saslServerProperties)
       .setSaslClientProperties(saslClientProperties)
       .setServerProperties(serverProperties)
       .build()).asJava
   }
 }
 
+// TODO: KAFKA-17631 - Convert SaslApiVersionsRequestTest to kraft
+@Disabled
```

Comment:
Same comment as above

Reply:
Addressed both comments. Thanks for the review.